lib/types.h

```diff
@@ -219,16 +195,17 @@ class Camera : public aiCamera {
      * transformation matrix.
      */
     Vec raster2cam(const aiVector2D& p, float w, float h) const {
-        return trafo_ * Vec(-delta_x_ * (1 - 2 * p.x / w),
-                            delta_y_ * (1 - 2 * p.y / h), -1);
+        aiVector3D v = trafo_ * aiVector3D(-delta_x_ * (1 - 2 * p.x / w),
```

Comment:
Yes, it will. Now we have to do it like that, since we don't have our own implementation of transformations.